engine-tests/src/test/java/org/terasology/persistence/WorldSerializerTest.java

```diff
@@ -50,7 +35,8 @@
     public void testNotPersistedIfFlagedOtherwise() throws Exception {
         EngineEntityManager entityManager = context.get(EngineEntityManager.class);
         EntityBuilder entityBuilder = entityManager.newBuilder();
-        WorldSerializer worldSerializer = new WorldSerializerImpl(entityManager, new PrefabSerializer(entityManager.getComponentLibrary(), entityManager.getTypeSerializerLibrary()));
+        WorldSerializer worldSerializer =
+                new WorldSerializerImpl(entityManager, new PrefabSerializer(entityManager.getComponentLibrary(), entityManager.getTypeSerializerLibrary()));
```

Comment:
I suggest extracting a local variable for the prefab serializer. This makes is easier to read and the line become significantly shorter.